src/containers/Dashboard.vue

```diff
@@ -246,6 +252,17 @@ export default Vue.extend({
     background-color: rgb(0, 0, 0); /* Fallback color */
     background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
   }
+
+  &-builtBy {
+    text-align: right;
+    font-size: 15px;
+    padding: 8px 10px;
+
+    a {
+      color: $medGray;
+      text-decoration: underline $medGray;
```

Comment:
I believe we want underline on hover. @einc can you confirm?

Reply:
Also I'm not sure if the color for "Built with .. by..." is exactly right. I can't seem to find the Figma file for this... @einc can you drop the Figma link?